### ISSUE TITLE
Question - What is updating functionality for?

### ISSUE THREAD

**Issue by rerobins** [nextcloud/fulltextsearch_elasticsearch#56]
https://github.com/nextcloud/fulltextsearch_elasticsearch/blob/a73d8fad6dc9dc7b015d11b0d7f9496fd36a0646/lib/Service/IndexService.php#L193

I'm not understanding the logic in line 193.  To me updating is when the content or the metadata changes, but that's not the case in this section as it specifically says don't use the update functionality if the metadata or content changes.

**Comment by daita** [comment]
Yes, this is not yet implemented, updating should only update few info (meta or content).

Right now, update is the same as full indexing.
